OntologyMapper/src/OntologyMapper.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <Version>$(VersionPrefix)</Version>
+    <Authors>Microsoft Corporation</Authors>
+    <Description>Support for converting from one DTDL-based Ontology to a different DTDL Based Ontology</Description>
+    <AssemblyVersion>0.1.0</AssemblyVersion>
+    <RepositoryUrl>https://github.com/Azure/opendigitaltwins-tools</RepositoryUrl>
+    <Copyright>Copyright (c) Microsoft Corporation.  All rights reserved.</Copyright>
+    <AssemblyName>Microsoft.SmartFacilities.OntologyMapper</AssemblyName>
+    <RootNamespace>Microsoft.SmartFacilities.OntologyMapper</RootNamespace>
+  </PropertyGroup>
+	
+  <ItemGroup>
+    <EmbeddedResource Include="Mappings\Mapped\Json\v0\BrickRec\mapped_json_v0_dtdlv2_Brick_1_3-REC_4_0.json" />
```

Comment:
Would it make sense to add these files as a wildcard? Seems like it would be easy for newcomers to miss